### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by orcunsagirsoy**
Footer component:

-  add component using mock data

-  add snapshot test

-  logo missing at the right side of the footer

footer

-footer refactored according to comments

image

### CODE REVIEW

File: src/components/Footer/index.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+.footer {
+  font-family: "Acme";
+  background: #590995;
+  color: white;
+  width: "100%";
+  font-size: 14px;
+  height: 14em;
+}
+
+/* .address {
```
Comment (betulerkoc) [inline src/components/Footer/index.css]: we should remove comment lines

File: src/components/Footer/index.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    phone1: "+229-955-5388-336",
+    phone2: "+229-955-5373-360",
+  },
+  email: {
+    email1: "office@mail.com",
+    email2: "support@mail.com",
+  },
+};
+const menuData = {
+  explanation: `Lorem Ipsum is simply dummy text of the printing and
```
Comment (Ammar-64) [inline src/components/Footer/index.jsx]: don't use lorem ipsum texts

File: src/containers/Home/index.jsx [hunk 3]
```diff
@@ -1,11 +1,13 @@
 import React from "react";
 import HowItWorks from "../../components/HowItWorks";
+import Footer from "../../components/Footer";
 
 export default function Home() {
   return (
     <div>
       <h1>This is Home Page!</h1>
       <HowItWorks />
+      <Footer />
```
Comment (Ammar-64) [inline src/containers/Home/index.jsx]: footer is universal it should be imported into app.jsx

File: src/components/Footer/index.jsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  explanation: `Lorem Ipsum is simply dummy text of the printing and 
+    typesetting industry. Lorem Ipsum has been the industry 
+    standard dummy text ever since the 1500s`,
+  menu: ["Terms of Service", "Contact Us", "About", "Recent Posts"],
+};
+
+export default function Footer() {
+  return (
+    <Container fluid className="footer">
+      <Row className="mainRow">
```
Comment (Ammar-64) [inline src/components/Footer/index.jsx]: why are you using to many rows inside each other

**Review by Ammar-64 — CHANGES_REQUESTED** [review]
you should pull dev and merge it to your branch before because there're some updates there needs to be applied like .eslintrc

File: src/components/Footer/index.jsx (hunk 4 above)
Comment (orcunsagirsoy) [reply src/components/Footer/index.jsx]: to put the information on separate rows

**Review by knargocer — COMMENTED** [review]
Dear Orçun,

I don't think you need these many Row and Col tags as well. I think you can even manage it with one row and four columns. You can manage their positions in the row by adding the three contact info items in one div and aligning them with the logo (maybe with display: flex).

Also, as we already have the About us, Contact us, etc buttons in the navbar I guess these are unnecessary in the footer.

Thank you.

File: src/App.jsx [hunk 5]
```diff
@@ -1,11 +1,15 @@
 import React from "react";
 import Home from "./containers/Home";
+import Footer from "./components/Footer";
+import i18n from "./i18n";
+import { I18nextProvider } from "react-i18next";
 
 function App() {
   return (
-    <div>
+    <I18nextProvider i18n={i18n}>
```
Comment (Ammar-64) [inline src/App.jsx]: where not using “I18nextProvider i18n={i18n}” where using [useTranslation hook ](https://react.i18next.com/latest/usetranslation-hook)

**Review by Ammar-64 — COMMENTED** [review]
Can you please fix the i18n thing

**Review by knargocer — APPROVED** [review]

File: src/containers/Home/index.jsx (hunk 3 above)
Comment (orcunsagirsoy) [reply src/containers/Home/index.jsx]: PTAL

File: src/components/Footer/index.jsx (hunk 4 above)
Comment (orcunsagirsoy) [reply src/components/Footer/index.jsx]: PTAL

File: src/App.jsx (hunk 5 above)
Comment (orcunsagirsoy) [reply src/App.jsx]: PTAL

**Review by Ammar-64 — APPROVED** [review]

**Comment by Ammar-64** [comment]
@orcunsagirsoy can you please update the branch, pull dev merge it with footer, fix the conflict and push it agian

**Comment by Ammar-64** [comment]
@orcunsagirsoy there are still some conflicts needs to be fixed

**Pull request merged**